docs/content/docs/connectors/datastream/elasticsearch.md

```diff
@@ -65,240 +61,91 @@ about how to package the program with the libraries for cluster execution.
 
 Instructions for setting up an Elasticsearch cluster can be found
 [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup.html).
-Make sure to set and remember a cluster name. This must be set when
-creating an `ElasticsearchSink` for requesting document actions against your cluster.
 
 ## Elasticsearch Sink
 
-The `ElasticsearchSink` uses a `TransportClient` (before 6.x) or `RestHighLevelClient` (starting with 6.x) to communicate with an
-Elasticsearch cluster.
-
 The example below shows how to configure and create a sink:
 
+[//]: # "TODO: update with quickstart example"
```

Comment:
Is something missing here?

Reply:
Nothing missing, just forgot to remove the comment :)